plugins/alerts/frontend/public/javascripts/countly.models.js

```diff
@@ -228,4 +121,235 @@
+                                alertName: alertsList[i].alertName || '',
+                                type: alertsList[i].alertDataSubType || '',
+                                condtionText: alertsList[i].compareDescribe || '',
+                                enabled: alertsList[i].enabled || false,
+                                createdByUser: alertsList[i].createdByUser || ''
+                            });
+                            /*eslint-enable */
+                        }
+                        
+                    
+                        if (countlyGlobal.plugins.indexOf("concurrent_users") < 0) {
+                            context.commit("setAll", tableData);
+                            context.commit("setCount", count);
+                            return;
+                        }
+                        CV.$.ajax({
+                            type: "GET",
+                            url: countlyCommon.API_PARTS.data.r, 
+                            dataType: "json",
+                            data: {
+                                app_id: countlyCommon.ACTIVE_APP_ID,
+                                method: "concurrent_alerts",
+                                preventGlobalAbort: true,
+                            },
+                        }).then(function(list) {
+                            for(var i = 0; i < list.length; i++) {
+                                tableData.push({
+                                    ...list[i],
+                                    _id: list[i]._id,
+                                    alertName: list[i].name,
+                                    appNameList: countlyGlobal.apps[list[i].app].name,
+                                    condtionText: list[i].condition_title,
+                                    enabled: list[i].enabled,
+                                    selectedApps: [list[i].app],
+                                    alertDataType: "online-users",
+                                    alertDataSubType: list[i].type,
+                                    compareType: list[i].def,
+                                    compareValue: list[i].users,
+                                    compareValue2: list[i].minutes,
+                                    alertValues: list[i].email,
+                                 });
+
+                            }
+                            context.commit("setAll", tableData);
+                            context.commit("setCount", count);
+                        })
+
+                        
+                        
+
+                    });
+                },
+            }
+        });
+        return countlyVue.vuex.Module("countlyAlerts", {
+            resetFn: getEmptyState,
+            getters: getters,
+            actions: actions,
+            mutations: mutations,
+            submodules: [tableResource]
+        });
+    };
+
+    countlyAlerts.defaultDrawerConfigValue = function () {
```

Comment:
for empty states, we use a factory object like such - https://github.com/Countly/countly-server/blob/e15f9dfbafde261d61ec07c545738897a1cf3185/plugins/vue-example/frontend/public/javascripts/countly.models.js#L5-L20
But its okay for now. In future, you can do it this way.